pex/bin/pex.py

```diff
@@ -325,6 +326,29 @@ def configure_clp_pex_environment(parser):
         "#!. This overrides the default behavior, which picks an environment Python "
         "interpreter compatible with the one used to build the PEX file.",
     )
+    group.add_argument(
+        "--shboot",
+        "--sh-boot",
+        "--bash-boot",
+        "--no-shboot",
+        "--no-sh-boot",
+        "--no-bash-boot",
+        dest="sh_boot",
+        default=False,
```

Comment:
If backwards compatibility was not a thing, what would you want this default to be?

Reply:
Probably the opposite, but backwards compatibility is a thing. Pex ~doesn't break the CLI API, it only adds features or fixes bugs. Pants can default to this if it wants of course. Pex can choose to do so only in Pex 3.

Reply:
Agreed. Would it help to create a tracking issue that records all the small changes we want to make?

Reply:
Not really. Afaict we = me and doing that now just takes away my limited time hacking getting needed things done. I'll definitely step back and do that when getting close to the Pex 3 push though.